docs/core/install/linux-package-mixup.md

```diff
@@ -50,25 +50,25 @@ Mixing .NET packages from two different sources will most likely lead to issues
 
 The solution to these problems is to use .NET from one package repository. Which repository to pick, and how to do it, varies by use-case and the Linux distribution.
 
-If your distribution provides .NET packages, it's recommended that you use that package repository instead of Microsoft's.
+If your distribution provides .NET packages, and you only need those packages, it's recommended that you use that package repository instead of Microsoft's.
```

Comment:
Should this information be included in the decision tree at https://github.com/dotnet/docs/pull/34234/files#diff-6592b7e6623c9ccf0f88d1bb413c25ee426fe0c6de36b162058c774157b46764R35 ?

If I need to run both my app and some other Microsoft app (e.g. powershell), I am going to find way too late about the feed mixup gotchas in the current decision tree.